src/main/java/com/aetherteam/aether_genesis/blockTileEntity/HolystoneFurnaceBlockEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.aetherteam.aether_genesis.blockTileEntity;
+
+import com.aetherteam.aether.blockentity.AbstractAetherFurnaceBlockEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.FurnaceMenu;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class HolystoneFurnaceBlockEntity extends AbstractAetherFurnaceBlockEntity {
+    public HolystoneFurnaceBlockEntity(BlockPos pPos, BlockState pBlockState) {
+        super(GenesisBlockEntityTypes.HOLYSTONE_FURNACE.get(), pPos, pBlockState, RecipeType.SMELTING);
+    }
+
+    protected Component getDefaultName() {
+        return Component.translatable("container.holystone_furnace");
```

Comment:
This should be `menu.aether.holystone_furnace` instead of `container.holystone_furnace`. Also make sure to add to AetherLanguageData "Holystone Furnace" for this.